doc/howtos/vulnerability-management.rst

```diff
@@ -0,0 +1,145 @@
+component version, and potentially running any proof-of-concept
+exploits or vulnerability checks distributed with the vulnerability
+report. It is possible that the Ostro OS security mechanisms, such as
+minimal configuration of system services, already prevent the
+vulnerability from directly affecting Ostro OS. If that’s the case, the
+criticality of the vulnerability will naturally be lower.
+
+If Ostro OS is affected by the vulnerability, the vulnerability details
+will determine its criticality. For example: remotely exploitable
+issues are the worst, since they jeopardize IoT devices directly, but
+if the vulnerability is locally exploitable by applications,
+criticality will be somewhat lower. Ostro OS vulnerability criticality
+is determined using `Common Vulnerability Scoring System <https://nvd.n
+ist.gov/cvss.cfm?calculator&adv&version=2>`_. Based on the result of
+the CVSS tool and the Ostro Project security team’s own vulnerability
+assessment, the security team takes action for getting the
+vulnerability fixed.
+
+If the vulnerability is considered to be critical, a typical response
+is adding a patch to ``meta-ostro-fixes`` repository, releasing the
+patch in ``ostro-os`` repository, reporting the bug to the meta-layer
+which provides the component in question, and sending out an Ostro OS
+security advisory. The patch either updates the component in question
+to a version which doesn’t have the vulnerability or backports the fix
+if moving to a new component version is not desired. If the component
+cannot be fixed quickly, the security team needs to consider disabling
+it from Ostro OS temporarily. If the security vulnerability is critical
+enough and it is estimated to put existing Ostro OS -based devices in
+serious danger, Ostro OS can make an unscheduled release to get the bug
+fix quickly to a wider audience.
+
+If the vulnerability is not critical and it’s expected that upstream
+meta-layer providing the component will handle it promptly, it is often
+enough to make sure the upstream meta-layer is aware of the issue, and
+then just wait for a fix to be delivered via the normal update mechanism
+into Ostro OS. It's understood that the default mechanism for how
+security fixes come to Ostro OS is via the upstream meta-layers.
+
+If the vulnerability is in Ostro OS core, the problem needs to be fixed
+in Ostro OS itself, and the upstream considerations are not necessary.
+
+Finally, the processing of the vulnerability leads to a (periodic) look
+at Ostro OS security architecture. The Ostro Project security team has
+to consider if Ostro OS security architecture needs be adapted to
+better protect against the vulnerability and others like it. This is
+continuous work and eventually lead to an Ostro OS security
+architecture that is well prepared to handle many vulnerability
+classes.
+
+Tracking and discussing the security vulnerability and the fix to it
+happens in Ostro OS issue tracker. The security vulnerabilities in the
```

Comment:
English weird on that first sentence.  Should probably be "Tracking and discussion of the security vulnerability and its fix will happen in the Ostro OS issue tracker."